src/web/App.mjs

```diff
@@ -298,13 +296,24 @@ class App {
      *
      * @param {boolean} [minimise=false] - Set this flag if attempting to minimise frames to 0 width
      */
-    initialiseSplitter(minimise=false) {
+    initialiseUI() {
+        if (window.innerWidth < this.breakpoint) {
+            this.setMobileUI();
+        } else {
+            this.setDesktopUI(false);
+        }
+    }
+
+    /**
+     * Set desktop splitters
+     */
+    setDesktopSplitter(minimise) {
         if (this.columnSplitter) this.columnSplitter.destroy();
         if (this.ioSplitter) this.ioSplitter.destroy();
 
         this.columnSplitter = Split(["#operations", "#recipe", "#IO"], {
             sizes: [20, 30, 50],
-            minSize: minimise ? [0, 0, 0] : [240, 310, 450],
+            minSize: minimise ? [0, 0, 0] : [20, 330, 310],
```

Comment:
Sure, either way works but to avoid confusion I'm setting #operations to 360, so they make up 1000 together as before. I understand it can look weird otherwise on first sight